chapter_multilayer-perceptrons/underfit-overfit_vn.md

```diff
@@ -43,7 +52,11 @@ In a large hospital system, we might access hundreds of thousands of medical rec
 When working with finite samples, we run the risk that we might discover *apparent* associations that turn out not to hold up when we collect more data.
 -->
 
-*dịch đoạn phía trên*
+Sự nguy hiểm là khi chúng ta huấn luyện các mô hình, chúng ta truy cập chỉ một mẫu dữ liệu nhỏ.
+Các tệp dữ liệu hình ảnh công cộng lớn nhất chứa khoảng một triệu hình ảnh.
```

Comment:
`dữ liệu công cộng` nghe tếu vãi :))

Reply:
> Thank bạn, bản dịch tốt rồi. Mình có nhận xét chung là bạn xài nhiều chữ `chúng ta` quá :)).
> 
> * Chữ `pattern` mình thấy dịch rõ thành `khuôn mẫu` dễ phân biệt với `sample-mẫu` hơn.
> * Chữ `phổ quát` mình không chắc có tương đương với `generalization` không?

Mình cũng băn khoăn khi dịch từ *general* thành *phổ quát*, một phần mình tham khảo trong glossary nên sử dụng. 

> `dữ liệu công cộng` nghe tếu vãi :))
Đồng ý luôn. Bạn có gợi ý gì cho mình ở chỗ này ko?

Reply:
mình suggest `tập dữ liệu công khai`

Reply:
@tranyenthy cảm ơn bạn đã thả tim gợi ý của mình. Chắc bạn chưa biết, cơ mà gợi ý nào mà bạn đồng ý thì có thể `commit` hoặc `add to batch` để commit 1 lần nhiều gợi ý luôn. Chỗ nào bạn chưa đồng ý thì cứ để đó phản biện hoặc chờ thêm ý kiến.

Reply:
oops. okay bạn. @lkhphuc